src/renderer/components/theme-settings/theme-settings.js

```diff
@@ -111,6 +112,7 @@ export default defineComponent({
         this.$t('Settings.Theme Settings.Base Theme.Black'),
         // Second group
         this.$t('Settings.Theme Settings.Base Theme.Nordic'),
+        this.$t('Settings.Theme Settings.Base Theme.Gruvbox Dark'),
```

Comment:
Insert this below 
https://github.com/FreeTubeApp/FreeTube/pull/4805/files#diff-a0480aa13261014e61fefded30097805ea7559cfe7b068b66b209ae896b17adaR120
```suggestion
```